examples/jsm/renderers/webgpu/WebGPUBackend.js

```diff
@@ -1071,9 +1062,9 @@ class WebGPUBackend extends Backend {
 
 	}
 
-	hasFeature( name ) {
+	async hasFeature( name ) {
```

Comment:
I think we could create a new `hasFeatureAsync` to be used in `detectSupportAsync` and leave `hasFeature` as `chained method` returning `this`, we can use just the internal adapter `this.adapter` for `hasFeature`. We will probably have to have an `warn` the user saying if `this.adapter` has not been initialized and suggest that he can use the `detectSupportAsync` function instead.

Reply:
Thanks! Applied the feedbacks. 👍